lib/slide_hub/username_url_constrainer.rb

```diff
@@ -0,0 +1,7 @@
+module SlideHub
+  class UsernameUrlConstrainer
+    def matches?(request)
```

Comment:
__[Reek]__

[UtilityFunction] SlideHub::UsernameUrlConstrainer#matches? doesn't depend on instance state (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->